src/core/Akka/Configuration/ConfigurationFactory.cs

```diff
@@ -55,6 +55,7 @@ public static Config ParseString(string hocon)
             return ParseString(hocon, null);
         }
 
+#if !ANDROID
```

Comment:
Is it only the configuration stuff that needed to be  `IFDEF`-ed out in order for this to work?